index.html

```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
+            <div class ="otherproject">
+                <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" >
+            <div class ="otherproject">
+                <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+                <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul id ="project2language">
+                    <li class ="language2item">html</li>
+                    <li class ="language2item" id ="bootstrap">bootstrap</li>
+                    <li class ="language2item">Ruby</li>
+                </ul>
+                <button class="seeproject2">See Project</button>
+            </div>
+        </section>
+        <section class="workdescription" ><div class ="otherproject">
+            <h2 class="otherprojectheading">Profesional Art Printing Data</h2>
+            <p class ="otherprojecttext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul id ="project2language">
+                <li class ="language2item">html</li>
+                <li class ="language2item" id ="bootstrap">bootstrap</li>
+                <li class ="language2item">Ruby</li>
+            </ul>
+            <button class="seeproject2">See Project</button>
+        </div>
+    </section>
+        
+    </div>
+    <div class="aboutsection" >
+        <section class ="about">
 
+            <h1 class="aboutheader"> About me</h1>
+            <p id ="aboutmetext">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button id="getresumebutton">Get My Resume</button>
```

Comment:
- [ ] It is best practice to ensure every HTML button has type property. Kindly update this button tag with a `type="button"`. Please ensure the button looks exactly like what you have on Figma design. Please remove border.

**Figma**
![image](https://user-images.githubusercontent.com/84156714/212474584-06e0e470-dcd4-4d24-a2c4-da8685997080.png)

Your Button 
![image](https://user-images.githubusercontent.com/84156714/212474607-96c84f0c-34cd-447b-a276-8287b62af5f9.png)